include/avif/avif.h

```diff
@@ -828,22 +823,19 @@ typedef struct avifImage
     avifImageItemProperty * properties; // NULL only if numProperties is 0.
     size_t numProperties;
 
-#if defined(AVIF_ENABLE_GAIN_MAP)
     // Gain map image and metadata. NULL if no gain map is present.
     // Owned by the avifImage and gets freed when calling avifImageDestroy().
     // gainMap->image->transformFlags is always AVIF_TRANSFORM_NONE.
     avifGainMap * gainMap;
```

Comment:
Optional: We can move `gainMap` before `properties` because `gainMap` was added first.

Reply:
Ack, I think it can stay as is.